src/dataflow/src/sink/kafka.rs

```diff
@@ -537,18 +568,27 @@ where
         s.metrics.messages_in_flight.set(in_flight as u64);
 
         if !ready_rows.is_empty() {
+            debug!("Rescheduling because ready_rows is not empty");
```

Comment:
debug! doesn't include tid by default, so this logline isn't going to make sense for an mz instance with multiple worker threads

Reply:
Also sink id in case the instance has multiple sinks